cmd/application_domain_add.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var appDomainAddCmd = &cobra.Command{
+	Use:     "add",
+	Aliases: []string{"create", "new"},
+	Short:   "Add a new domain for your application.",
+	Args:    cobra.MinimumNArgs(2),
+	Example: "civo app domain add APP-NAME your-app-name.example.com",
+	Run: func(cmd *cobra.Command, args []string) {
+		utility.EnsureCurrentRegion()
+
+		client, err := config.CivoAPIClient()
+		if regionSet != "" {
+			client.Region = regionSet
+		}
+		if err != nil {
+			utility.Error("Creating the connection to Civo's API failed with %s", err)
+			os.Exit(1)
+		}
+
+		findApp, err := client.FindApplication(args[0])
+		if err != nil {
+			utility.Error("App %s", err)
+			os.Exit(1)
+		}
+
+		application := &civogo.UpdateApplicationRequest{
```

Comment:
Some validation to see if it's a valid domain/hostname would be nice

Reply:
Can we validate it this way? It uses a regular expression: https://github.com/asaskevich/govalidator/blob/f21760c49a8d/validator.go#L675

Reply:
Yes, done.

Reply:
Sorry, I don't see those changes yet.

Reply:
You can find those in the latest commit now.